### DISCUSSION TITLE
Container doesnt start

### DISCUSSION THREAD

**Discussion by Blueforcer** [mbround18/valheim-docker#506] [Help!]
Edit:
Just found a similat issue but without a solution
https://github.com/mbround18/valheim-docker/discussions/495

Hello!
i just tried to install the container via docker compose.
After Boot i get this log, and the container stops.
Did i missed something?

´´´
[Valheim][root] : ###########################################################################,
[Valheim][root] : Valheim Server - Thu 13 Jan 2022 03:26:00 PM ,
[Valheim][root] : Initializing your container...,
[Valheim][root] : You must be in development. Good luck!,
[Valheim][root] : ###########################################################################,
[Valheim][root] : Switching UID and GID,
[Valheim][root] : usermod: no changes,
[Valheim][root] : ,
[Valheim][root] : Auto Update Enabled...,
[Valheim][root] : Auto Update Schedule: 0 1 * * *,
[Valheim][root] : Auto Backup Enabled...,
[Valheim][root] : Auto Backup Schedule: */15 * * * *,
[Valheim][root] : Setting up file systems,
[Valheim][root] : Navigating to steam home...,
[Valheim][root] : Launching as steam...,
[Valheim][steam]: ###########################################################################,
[Valheim][steam]: Valheim Server - Thu 13 Jan 2022 03:26:00 PM ,
[Valheim][steam]: STEAM_UID 1000 - STEAM_GUID 1000,
[Valheim][steam]: Installing Valheim via odin 1.4.3...,
[Valheim][steam]: ###########################################################################,
[Valheim][steam]: Variables loaded.....,
[Valheim][steam]: Port: 2456,
[Valheim][steam]: Name: Sozialamt,
[Valheim][steam]: World: Dedicated,
[Valheim][steam]: Public: 1,
[Valheim][steam]: Password: (REDACTED),
[Valheim][steam]: Running Install...,
[ODIN][INFO]  - Installing 896660 to /home/steam/valheim,
[ODIN][INFO]  - steamcmd found in path,
[ODIN][INFO]  - Exited with http code: 1,
[Valheim][steam]: Initializing Variables....,
cp: cannot stat '/home/steam/steamcmd/linux64/steamclient.so': No such file or directory,
[ODIN][ERROR] - Failed to find server executable! Please run `odin install`,
```

**Comment by mbround18** [comment]
@Blueforcer I have seen this error a couple times, can you provide what version of docker you are utilizing? 

Ref: #501

**Reply by mbround18** [reply]
You might have given me recourse to get more information :)
